rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IOperationService.java

```diff
@@ -51,6 +52,11 @@
     Object execute(final OperationChain opChain);
 
     @POST
+    @Path("/operation")
```

Comment:
Might it be better to change around the paths for these operations now we're adding the "execute single operation" option?

Something like:

Execute operation chain: /graph/doOperation/chain
Execute single operation: /graph/doOperation

?